src/arrays.js

```diff
@@ -8,34 +8,59 @@ const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
   // based off http://underscorejs.org/#each
+  for (let i = 0; i < elements.length; i++) cb(elements[i], i);
 };
 
 const map = (elements, cb) => {
   // Produces a new array of values by mapping each value in list through a transformation function (iteratee).
   // Return the new array.
+  return elements.map((elem, i) => cb(elem));
```

Comment:
Using the native array methods isn't allowed on this lab. Try reimplementing this without it.